feature/gribi/ate_tests/leader_failover_test/leader_failover_test.go

```diff
@@ -250,7 +250,9 @@ func TestLeaderFailover(t *testing.T) {
 	}
 
 	t.Run("SINGLE_PRIMARY/PERSISTENCE=DELETE", func(t *testing.T) {
-
+		if *deviations.GRIBIPreserveOnly {
+			t.Skip()
```

Comment:
Can we add a skip reason?  I see other tests we have used t.Skip("Skipping due to --deviations_gribi_preserve_only").  This makes logs easier to read.

Reply:
thanks!, addressed